atlasdb-client/src/integrationInput/java/com/palantir/atlasdb/table/description/generated/SchemaApiTestV2Table.java

```diff
@@ -0,0 +1,273 @@
+        rangeRequest = rangeRequest.getBuilder().retainColumns(colSelection).build();
+        Preconditions.checkArgument(rangeRequest.getColumnNames().size() <= 1,
+                "Must not request columns other than Column2.");
+        return BatchingVisitableView.of(t.getRange(tableRef, rangeRequest))
+                .immutableCopy()
+                .stream()
+                .map(entry -> SchemaApiTestTable.SchemaApiTestRowResult.of(entry))
+                .collect(Collectors.toMap(
+                     entry -> entry.getRowName().getComponent1(), 
+                     SchemaApiTestTable.SchemaApiTestRowResult::getColumn2));
+    }
+
+    /**
+     * Delete all columns for specified row components. */
+    public void deleteRow(String component1) {
+        SchemaApiTestTable.SchemaApiTestRow row = SchemaApiTestTable.SchemaApiTestRow.of(component1);
+        byte[] rowBytes = row.persistToBytes();
+        Set<Cell> cells = new HashSet<>();
+        cells.add(Cell.create(rowBytes, PtBytes.toCachedBytes("c")));
+        cells.add(Cell.create(rowBytes, PtBytes.toCachedBytes("d")));
+        t.delete(tableRef, cells);
+    }
+
+    /**
+     * Delete the value at column Column1 (if it exists) for the specified row-key. */
+    public void deleteColumn1(String component1) {
+        SchemaApiTestTable.SchemaApiTestRow row = SchemaApiTestTable.SchemaApiTestRow.of(component1);
+        byte[] rowBytes = row.persistToBytes();
+        Set<Cell> cells = ImmutableSet.of(Cell.create(rowBytes, PtBytes.toCachedBytes("c")));
+        t.delete(tableRef, cells);
+    }
+
+    /**
+     * Delete the value at column Column2 (if it exists) for the specified row-key. */
+    public void deleteColumn2(String component1) {
+        SchemaApiTestTable.SchemaApiTestRow row = SchemaApiTestTable.SchemaApiTestRow.of(component1);
+        byte[] rowBytes = row.persistToBytes();
+        Set<Cell> cells = ImmutableSet.of(Cell.create(rowBytes, PtBytes.toCachedBytes("d")));
+        t.delete(tableRef, cells);
+    }
+
+    /**
+     * Takes the row-keys and a value to be inserted at column Column1. */
+    public void putColumn1(String component1, Long column1) {
+        SchemaApiTestTable.SchemaApiTestRow row = SchemaApiTestTable.SchemaApiTestRow.of(component1);
+        t.put(tableRef, ColumnValues.toCellValues(ImmutableMultimap.of(row, SchemaApiTestTable.Column1.of(column1))));
+    }
+
+    /**
+     * Takes a function that would update the value at column Column1, for the specified row
+     * components. No effect if there is no value at that column. Doesn't do an additional
+     * write if the new value is the same as the old one. */
+    public void updateColumn1(String component1, Function<Long, Long> processor) {
+        Optional<Long> result = getColumn1(component1);
+        if (result.isPresent()) {
+            Long newValue = processor.apply(result.get());
+            if (newValue != result.get()) {
```

Comment:
How does the reference comparison work if the value is string or blob type? We probably want `Objects.equals(new, old)`